<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>662</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="Country_Table">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>40</y>
     <width>261</width>
     <height>141</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Country Code</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Country Name</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="Add_Country_Button">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>200</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Country</string>
   </property>
  </widget>
  <widget class="QComboBox" name="Category_ComboBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>40</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Mens</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Womens</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="Format_ComboBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>90</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>T20I</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ODI</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Test</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="Category_Label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Category:</string>
   </property>
  </widget>
  <widget class="QLabel" name="Format_Label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Format:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Add_Team_Button">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>300</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Team</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Password_Entry">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>140</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="Password_Label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>150</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Password:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Confirm_Password_Entry">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>200</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="Confirm_Password_Label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Confirm Password:</string>
   </property>
  </widget>
  <widget class="QLabel" name="Error_Label">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>270</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #f00;</string>
   </property>
   <property name="text">
    <string>Please specify Country, Password</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
